pkg/controller/core.oam.dev/v1alpha2/application/application_controller_test.go

```diff
@@ -608,20 +582,12 @@ spec:
 		Expect(k8sClient.Get(ctx, appKey, curApp)).Should(BeNil())
 		Expect(curApp.Status.Phase).Should(Equal(common.ApplicationRunning))
 
-		By("Check AppConfig and trait created as expected")
-		appContext := &v1alpha2.ApplicationContext{}
-		Expect(k8sClient.Get(ctx, client.ObjectKey{
-			Namespace: app.Namespace,
-			Name:      app.Name,
-		}, appContext)).Should(BeNil())
 		appRevision := &v1beta1.ApplicationRevision{}
 		Expect(k8sClient.Get(ctx, client.ObjectKey{
 			Namespace: app.Namespace,
 			Name:      curApp.Status.LatestRevision.Name,
 		}, appRevision)).Should(BeNil())
 
-		Expect(appContext.Spec.ApplicationRevisionName).Should(Equal(appRevision.Name))
```

Comment:
Do we have any way to check the revision name? Maybe we need to check the real resource?

Reply:
Yes, it makes sense. Let me find a way to check created resources.

Reply:
Fix by checking the affiliated resource tracker with an expected revision name.
We will use resource tracker to check everything checked by appCtx before.